packages/core/src/libraries/hook/index.ts

```diff
@@ -54,6 +37,62 @@ export const createHookLibrary = (queries: Queries) => {
     hooks: { findAllHooks, findHookById },
   } = queries;
 
+  // Send webhook requests with given payloads to the matching hooks. Log the response and error is any.
```

Comment:
- should be in the JSDoc format?
- it's unclear for "log the response...", better to have "create an audit log entry for..."
- "is any" typo?